src/hooks/useSelectedTransactionsActions.ts

```diff
@@ -0,0 +1,117 @@
+import {useMemo} from 'react';
+import * as Expensicons from '@components/Icon/Expensicons';
+import {useMoneyRequestReportContext} from '@components/MoneyRequestReportView/MoneyRequestReportContext';
+import {deleteMoneyRequest, unholdRequest} from '@libs/actions/IOU';
+import {exportReportToCSV} from '@libs/actions/Report';
+import Navigation from '@libs/Navigation/Navigation';
+import {getIOUActionForTransactionID, getOriginalMessage, isDeletedAction, isMoneyRequestAction} from '@libs/ReportActionsUtils';
+import {canDeleteCardTransactionByLiabilityType, canDeleteTransaction} from '@libs/ReportUtils';
+import {getTransaction, isOnHold} from '@libs/TransactionUtils';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type {OriginalMessageIOU, Report, ReportAction, Session} from '@src/types/onyx';
+import useLocalize from './useLocalize';
+
+const HOLD = 'HOLD';
```

Comment:
Wait what why this and not take it from the const secondary actions?

Reply:
Yeah, but `SECONDARY_ACTIONS` includes `HOLD` and doesn't includes `UNHOLD` and `PRIMARY_ACTIONS` includes `UNHOLD and doesn't includes `HOLD` 😅 And this value is ignored because we are interested only in actions

Reply:
So I added those constants above in order to don't mix `SECONDARY_ACTIONS` with `PRIMARY_ACTIONS` especially since those values were defined for usage in header, not in selection mode